publish-to-schedule.php

```diff
@@ -234,7 +234,7 @@ function pts_findNextSlot($post,$changePost = False){
 	global $wpdb;
 	global $table_prefix;
 	global $pts_debug;
-	
+	global $pts_options;

```

Comment:
Need to declare  `$pts_options` as global so results of the `get_option(PTS_OPTION_NAME)` call will be available elsewhere.

Reply:
Thank you for your help fixing this.